docs/en/stack/ml/df-analytics/evaluatedf-api.asciidoc

```diff
@@ -0,0 +1,100 @@
+[role="xpack"]
+[[ml-dfanalytics-evaluate]]
+== Evaluating {dfanalytics}
+
+Using the {dfanalytics} features to gain insights from a data set is an 
+iterative process. You might need to experiment with different analyses, 
+parameters, ways to transform data before you arrive to a result that satisfies 
+your use case. A valuable companion to this process is the 
+{ref}/evaluate-dfanalytics.html[{evaluatedf-api}] that allows you to evaluate 
+the {dfanalytics} performance against a marked up data set. It helps to 
+understand error distributions and to know the points where the {dfanalytics} 
+model performing well or less trustworthily.
+
+{evaluatedf-api-cap} is designed for providing a general evaluation mechanism 
+for the different kinds of {dfanalytics}. That means that every type of analysis 
+can be evaluated by using one of the evaluation types that the {evaluatedf-api} 
+offers. For example, the results of an {oldetection} analysis can be evaluated 
+by using {binarysc} through the {evaluatedf-api}.
+
+To evaluate an algorithm with this API, you need to annotate your index that 
+contains the results of the analysis with a field that marks each {datafame} row 
+with the ground truth (for example, in case of {oldetection}, whether the given 
+data point really is an outlier or not). The {evaluatedf-api} will evaluate the 
+performance of the {dfanalytics} against this manually provided ground truth.
+
+[discrete]
+[[ml-dfanalytics-binary-soft-classification]]
+=== {binarysc-cap} evaluation
+
+{oldetection-cap} essentially classifies each data point in a data set into one 
```

Comment:
While it is great that prior to this we provide an example of outlier detection, I think we should not use outlier detection terminology here. Binary soft classification can be used for other types of analyses too. For instance, we could start with:

```This evaluation type is suitable for analyses which calculate a probability that each data point in a data set is a member of a class or not. ``` 

or something along those lines.